src/component/term/TermMetadata.tsx

```diff
@@ -81,6 +83,10 @@ export class TermMetadata extends React.Component<TermMetadataProps, TermMetadat
         this.setState({assignmentsCount});
     };
 
+    private setCommentsCount = (commentsCount: number) => {
```

Comment:
I don't think it should be `private`, since it is passed to another component  as a callback. But there are no hard rules in this regard in TypeScript...

Reply:
It is just a variable, no? What sort of rules/recommendations you refer to?

Reply:
Nevermind, as I said, I don't know of any guidelines or rules against this. It just feels weird to me to have a private member (it is indeed a field, but I always see these property initializers as methods, because that's what they would be in Java) that is essentially exported from a class...
Keep it as it is.